packages/block-editor/src/hooks/duotone.js

```diff
@@ -265,7 +265,7 @@ function useDuotoneStyles( {
 
 	const isValidFilter = Array.isArray( colors ) || colors === 'unset';
 
-	useStyleOverride(
+	usePrivateStyleOverride(
```

Comment:
Tiny nit about consistency:

I see Duotone and block style variations have been switched to use `usePrivateStyleOverride` instead of `useStyleOverride`. Is there a reason to switch these internal-to-the-block-editor instances to use the private one? If so, would it be worth updating the code in `layout-child.js`, `layout.js`, `position.js` and `style.js`, too? If not, then I was wondering if it would be better for these ones to use `useStyleOverride` instead of the private hook 🤔

Reply:
All good, I typed before thinking that comment through! 😄

Reply:
I believe the block style variations and duotone use the private version as they rely on the restricted parameters we're deliberately omitting from the public `useStyleOverride` e.g. `clientId`, `__unstableType` etc.

Reply:
Thanks for confirming!